07LessonSeven/homework/css/style.css

```diff
@@ -0,0 +1,70 @@
+body {
+   font-family: Arial;
+   width: 77%;
+}
+
+h1 {
+   margin-left: 125px;
+}
+
+#subheader {
+    background-color: #303030;
+    color: white;
+    padding: 5px;
+}
+
+#subheader > p {
+   margin-left: 125px;
+   font-size: 12pt;
+   word-spacing: 30px;
+}
+
+.box1 {
+   line-height: 25px;
+   background-color: #669940;
+   height: 260px;
+   width: 600px;
+   float: left;
+   margin-left: 125px;
+   color: white;
+}
+
+.text1  {
+   padding-top: 45px;
+   margin-left: 30px;
+   margin-right: 30px;
+}
+
+.text1 button {
+   background-color: #669940;
+   color: white;
+   border: 2px solid white;
+   margin-left: 340px;
+   font-size: 13pt;
+   padding: 15px;
+}
+
+.box2 {
+   line-height: 25px;
+   width: 600px;
+   height: 260px;
+   float: left;
+   background-color: #303030;
+   margin-right: 125px;
+   color: white;
+}
+
```

Comment:
the two boxes look great, but their widths are making it so they stack on my screen. try making them smaller or working with %s.